Dockerfile

```diff
@@ -1,25 +1,39 @@
-FROM python:3
-MAINTAINER Bryce Beuerlein <bryce.beuerlein@cisa.dhs.gov>
-ENV PCA_HOME="/home/pca" \
-    PCA_CON_SRC="/usr/src/pca-assessment"
+ARG GIT_COMMIT=unspecified
+ARG GIT_REMOTE=unspecified
+ARG VERSION=unspecified
 
-RUN groupadd --system pca && useradd --system --gid pca pca
+FROM python:3.7-alpine
 
-RUN apt-get update && \
-apt-get install --no-install-recommends -y \
-at &&\
-apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ARG GIT_COMMIT
+ARG GIT_REMOTE
+ARG VERSION
 
-RUN mkdir ${PCA_HOME} && chown pca:pca ${PCA_HOME}
-VOLUME ${PCA_HOME}
+LABEL git_commit=${GIT_COMMIT}
```

Comment:
The `{}` are only necessary in cases such as `RUN chmod +x ${GOPHISH_TOOLS_SRC}/var/getenv` where they are necessary to separate the variable name from its surroundings.  You could remove most of them in this file.

Reply:
All of those curly braces are coming from the [`skeleton-docker` Dockerfile](https://github.com/cisagov/skeleton-docker/blob/develop/Dockerfile), so @felddy, you should include this in your list of changes to make in there.  

In the meantime, I made those changes for this project with https://github.com/cisagov/gophish-tools/pull/7/commits/ce5f17c205b65e0c10df6654713dcb07d39ae710 and verified that I was still able to successfully build the Docker image.  Thanks @jsf9k catching that!